addons/mass_mailing_sms/tests/common.py

```diff
@@ -46,12 +46,14 @@ def assertSMSTraces(self, recipients_info, mailing, records,
           generated;
```

Comment:
> commit
"in a previous commit"
Might be good to use the commit hash once it's set in stone if possible.

Reply:
Maybe add something like "in this pull request" then, so we know we can reference the "closes [PR number]", or at least that the commit will be next to this one.